app/Http/Controllers/RankingController.php

```diff
@@ -229,10 +243,19 @@ public function kudosu()
         $page = min(get_int(request('page')) ?? 1, $maxPage);
 
         $scores = User::default()
-            ->with('country')
             ->orderBy('osu_kudostotal', 'desc')
             ->paginate(static::PAGE_SIZE, ['*'], 'page', $page, $maxResults);
 
+        if (is_json_request()) {
+            return json_collection(
```

Comment:
this should be inside `['ranking' => json_collection(...)]` just like other responses here (also new responses shouldn't return array in the first place)